### ISSUE TITLE
Missing import in BugsnagCrashSentry.h

### ISSUE THREAD

**Issue by erikkerber** [bugsnag/bugsnag-cocoa#971]
### Describe the bug

When compiling from source, without the provided Xcode project, compilation may fail due to a missing import in BugsnagCrashSentry.h

A reference to `BSC_KSCrashType` will fail compilation

### Steps to reproduce

1. Build Bugsnag, possibly with an alternate build s system such as Bazel where file inclusions are included alternatively to an Xcode project.
2. Build

```bash

1): worker failed: error executing command bazel-out/host/bin/external/build_bazel_rules_swift/tools/worker/worker swiftc @bazel-out/ios-x86_64-min12.0-applebin_ios-ios_x86_64-dbg/bin/src/Apps/Foo/Foo.swiftmodule-0.params
<module-includes>:4:9: note: in file included from <module-includes>:4:
#import "../../../../../../external/com_github_bugsnag_bugsnag_cocoa/Bugsnag/BugsnagCrashSentry.h"
        ^
/private/var/tmp/_bazel_erik/16c37bde558b14a272b5936eb51c4a21/execroot/foo/bazel-out/ios-x86_64-min12.0-applebin_ios-ios_x86_64-dbg/bin/external/com_github_bugsnag_bugsnag_cocoa/Bugsnag.modulemaps/../../../../../../external/com_github_bugsnag_bugsnag_cocoa/Bugsnag/BugsnagCrashSentry.h:33:4: error: expected a type
- (BSG_KSCrashType)mapKSToBSGCrashTypes:(BugsnagErrorTypes *)errorTypes;
   ^
src/Apps/Flagship/Sources/ApplicationDelegate.swift:16:8: error: could not build Objective-C module 'Bugsnag'
import Bugsnag
       ^
<foo> failed to build
```

### Environment
* Bugsnag version: 6.5.0
* CocoaPods version: N/A
* Carthage version (if any): N/A
* iOS/tvOS/macOS version(s): N/A
* Simulator or physical device: N/A
* Xcode version: 12.3.0
* Swift version (if applicable): 5.1

### Example Repo 

- [ ] Create a minimal repository that can reproduce the issue
- [ ] Link to it here:

### Example code snippet 

```
# (Insert code sample to reproduce the problem)
```

<details><summary>Error messages:</summary>

```

```
</details>

**Comment by mattdyoung** [comment]
Hi @erikkerber 

Thanks for the report and the PR. We'll aim to get this merged and released soon.

**Comment by erikkerber** [comment]
Thanks!

**Comment by yousif-bugsnag** [comment]
This is now fixed in [v6.6.0](url).
